timed/projects/filters.py

```diff
@@ -107,9 +114,7 @@ def filter(self, qs, value):
         )
         qs = qs.annotate(frequency=Count("reports")).order_by("-frequency")
         # limit number of results to given value
-        qs = qs[: int(value)]
-
-        return qs
+        return qs[: int(value)]
```

Comment:
This whole filter is kind of smelly - should be cleaned up in a separate PR:

* It's not a filter only
* It does implicit ordering
* It does implicit "pagination"

To make it work properly, this will need corresponding fixes in the frontend as well I think